src/pages/training/TrainingPage.svelte

```diff
@@ -84,7 +84,7 @@
         </p>
       </div>
     {:else}
-      <div cla54s="w-3/4 text-primarytext">
+      <div class="w-3/4 text-primarytext">
```

Comment:
This fix leads to a minor visual change as the width setting now takes effect. As the content is fairly narrow it only takes effect with a narrower window and seems like it was always intended.

Reply:
Funny, nicely done :)